static/js/pages/applications/ApplicationDashboardPage.js

```diff
@@ -72,121 +93,344 @@ export class ApplicationDashboardPage extends React.Component<Props, State> {
+              {formatReadableDateFromStr(submission.submitted_date)}
+            </div>
+          ) : (
+            <div>
+              <span className="label">Deadline: </span>
+              {formatReadableDateFromStr(step.due_date)}
+            </div>
+          )}
         </div>
-        <div className="col-12">
-          <h3>Application Review</h3>
+      </React.Fragment>
+    )
+  }
+
+  SUBMISSION_STEP_RENDERERS: { [ValidAppStepType]: Function } = {
+    [SUBMISSION_VIDEO]: this.renderVideoInterviewStep,
+    [SUBMISSION_QUIZ]:  this.renderQuizStep
+  }
+
+  renderReviewStep = (submission: ?ApplicationSubmission): React$Element<*> => {
+    let status
+    if (submission) {
+      if (!submission.review_status_date) {
+        status = "Pending"
+      } else if (submission.review_status === REVIEW_STATUS_REJECTED) {
+        status = "Rejected"
+      } else {
+        status = "Approved"
+      }
+    }
+
+    return (
+      <React.Fragment>
+        <h3>Application Review</h3>
+        {status && (
+          <div className="col-12 col-sm-6 status-text">
+            <span className="label">Status: </span>
+            {status}
+          </div>
+        )}
+      </React.Fragment>
+    )
+  }
+
+  renderApplicationDetailSection = (application: Application) => {
+    const { allApplicationDetail, currentUser, openDrawer } = this.props
+
+    if (
+      !allApplicationDetail ||
+      !allApplicationDetail[String(application.id)]
+    ) {
+      return null
+    }
+    const applicationDetail = allApplicationDetail[String(application.id)]
+    const appStepSubmissions = this.getAppStepSubmissions(
+      applicationDetail.run_application_steps,
+      applicationDetail.submissions
+    )
+
+    const profileFulfilled =
+      !!currentUser.profile && currentUser.profile.is_complete
+    const profileRow = this.renderDetailRow(
+      <React.Fragment>
+        <h3>Profile Information</h3>
+        <div className="col-12 col-sm-6 status-text">
+          {currentUser.profile && currentUser.profile.updated_on ? (
+            <div>
+              <span className="label">Last Updated: </span>
+              {formatReadableDateFromStr(currentUser.profile.updated_on)}
+            </div>
+          ) : null}
         </div>
-        <div className="col-12">
-          <h3>Payment</h3>
+        <div className="col-12 col-sm-5 text-sm-right">
           <a
             className="btn-link"
-            onClick={() => {
-              setDrawerState(PAYMENT)
-              setDrawerMeta({
-                applicationId
-              })
-              setDrawerOpen(true)
-            }}
+            onClick={R.partial(openDrawer, [{ type: PROFILE_VIEW }])}
           >
-            Make a Payment
+            View/Edit Profile
           </a>
         </div>
+      </React.Fragment>,
+      "profile",
+      profileFulfilled
+    )
+
+    const resumeFulfilled = !!applicationDetail.resume_upload_date
```

Comment:
`!!applicationDetail.resume_upload_date || applicationDetail.linkedin_url`

Reply:
This is just unfortunate naming. `resume_upload_date` is updated if a used uploads a resume _or_ adds a linkedin URL. If that field has a value, one of the two items were added and the requirement is fulfilled